shared/gh/api/gh.api.util.js

```diff
@@ -455,26 +455,30 @@ define(['exports', 'moment', 'sinon', 'bootstrap-notify'], function(exports, mom
         require(['text!gh/partials/admin-module-item.html',
                  'text!gh/partials/admin-modules.html',
                  'text!gh/partials/calendar.html',
+                 'text!gh/partials/editable-parts.html',
```

Comment:
I've been thinking it might be nice to come up with an alternative for defining partials. Nothing for now though.